implementations/micrometer-registry-otlp/src/main/java/io/micrometer/registry/otlp/OtlpMeterRegistry.java

```diff
@@ -82,13 +90,18 @@ private OtlpMeterRegistry(OtlpConfig config, Clock clock, HttpSender httpSender)
         this.config = config;
         this.httpSender = httpSender;
         this.resource = Resource.newBuilder().addAllAttributes(getResourceAttributes()).build();
+        this.isDeltaAggregationTemporality = config
+                .aggregationTemporality() == AggregationTemporality.AGGREGATION_TEMPORALITY_DELTA;
+        this.otlpAggregationTemporality = AggregationTemporality.mapToOtlp(config.aggregationTemporality());
```

Comment:
I would be more happy to commit to the suggestion so that it reflects the contribution but unfortunately Github doesn't let me commit the suggestion. So, I am going to fix with a new commit.